### PR TITLE
Added documentation steps for setup of Kubernetes dashboard.

### PR DESCRIPTION
**Pull request by karok2m**

### CODE REVIEW

File: docs/deploy/howto-deploy-aks.md [hunk 1]
```diff
@@ -514,34 +514,80 @@ Command line argument key-value pairs can be specified with:
 
 ### AKS
 
-All of microservices of Industrial IoT solution are deployed to an AKS Kubernetes cluster. The deployment
-creates `industrial-iot` namespace where all microservices are running. To provide connection details for
-all Azure resources created by `Microsoft.Azure.IIoT.Deployment`, we created `industrial-iot-env` secret,
-which is then consumed by deployments.
+All of microservices of Industrial IoT solution are deployed to an AKS Kubernetes cluster.
+
```
Comment (monikavar) [inline docs/deploy/howto-deploy-aks.md]: All cloud microservices of Industrial IoT platform...

File: docs/deploy/howto-deploy-aks.md [hunk 2]
```diff
@@ -514,34 +514,80 @@ Command line argument key-value pairs can be specified with:
...
+#### Resource Definitions
+
+The deployment creates `industrial-iot` namespace where all microservices are running. To provide connection
+details for all Azure resources created by `Microsoft.Azure.IIoT.Deployment`, we created
+`industrial-iot-env` secret, which is then consumed by deployments.
 
 To see YAML files of all Kubernetes resources that are created by the application check
```
Comment (monikavar) [inline docs/deploy/howto-deploy-aks.md]: by the application, please check:

File: docs/deploy/howto-deploy-aks.md [hunk 3]
```diff
@@ -514,34 +514,80 @@ Command line argument key-value pairs can be specified with:
...
+    download and install `kubectl`:
+
+    ```bash
+    az aks install-cli
+    ```
+
+    More details about the command and its optional parameters can be found [here](https://docs.microsoft.com/cli/azure/aks?view=azure-cli-latest#az-aks-install-cli).
+
+2. Get access credentials for a managed Kubernetes cluster. You would need name of the resource group
+    where AKS resource is and the name of AKS resource itself:
```
Comment (monikavar) [inline docs/deploy/howto-deploy-aks.md]: You would need the name of the resource group containing AKS resource and the name of the AKS resource:

File: docs/deploy/howto-deploy-aks.md [hunk 4]
```diff
@@ -514,34 +514,80 @@ Command line argument key-value pairs can be specified with:
...
+3. Create `ClusterRoleBinding` that will bind `ServiceAccount` of Kubernetes dashboard to `cluster-admin`
+    role. By default, the Kubernetes dashboard is deployed with minimal read access and displays RBAC access
+    errors.
+
+    ```bash
+    kubectl create clusterrolebinding kubernetes-dashboard --clusterrole=cluster-admin --serviceaccount=kube-system:kubernetes-dashboard
+    ```
+
+4. Show the dashboard for a Kubernetes cluster in a web browser. You would need name of the resource group
+    where AKS resource is and the name of AKS resource itself:
```
Comment (monikavar) [inline docs/deploy/howto-deploy-aks.md]: "Open" instead of "Show " IMO

Comment (monikavar) [reply docs/deploy/howto-deploy-aks.md]: Second line same as above comment.

File: docs/deploy/howto-deploy-aks.md (hunk 1 above)
Comment (karok2m) [reply docs/deploy/howto-deploy-aks.md]: Done.

File: docs/deploy/howto-deploy-aks.md (hunk 2 above)
Comment (karok2m) [reply docs/deploy/howto-deploy-aks.md]: Done.

File: docs/deploy/howto-deploy-aks.md (hunk 3 above)
Comment (karok2m) [reply docs/deploy/howto-deploy-aks.md]: Done.

File: docs/deploy/howto-deploy-aks.md (hunk 4 above)
Comment (karok2m) [reply docs/deploy/howto-deploy-aks.md]: Done.

**Pull request merged**